proto/nibiru/perp/v2/tx.proto

```diff
@@ -328,3 +332,34 @@ message MsgDonateToEcosystemFund {
 }
 
 message MsgDonateToEcosystemFundResponse {}
+
+// -------------------------- AllocateEpochRebates --------------------------
+message MsgAllocateEpochRebates {
+  string sender = 1;
+
+  // rebates to allocate
+  repeated cosmos.base.v1beta1.Coin rebates = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
```

Comment:
I think that's a good idea. 

We just have to make sure that we're not able to load past epochs because that creates a race condition where a user withdraws from a past epoch, then we load that past epoch, and the user misses out on that additional amount because we delete their accounting entry. 

Or we'd have to introduce complex accounting machinery to track when a a past epoch received additional load after a user withdraws, which I would prefer not to do.